src/components/search/data/searchLoader.js

```diff
@@ -1,5 +1,6 @@
+import { generatePath, redirect } from 'react-router-dom';
 import { getConfig } from '@edx/frontend-platform/config';
-import { ensureAuthenticatedUser } from '../../app/routes/data';
+import { ensureAuthenticatedUser } from '../../app/routes/data/utils';
```

Comment:
tiny nit: the `utils` at the end shouldn't be needed; removing the `/utils` here and in the associated test's import and mock works as well (verified in local checkout of this PR).

Reply:
[update/edit]: Ah, I see now... ignore above comment. Its still a dependency cycle within `createAppRouter`. Can leave it as is for now. False alarm!